src/main/java/org/zalando/nakadi/controller/TopicsController.java

```diff
@@ -0,0 +1,54 @@
+package org.zalando.nakadi.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.exceptions.ForbiddenAccessException;
+import org.zalando.nakadi.exceptions.runtime.NoTopicFoundException;
+import org.zalando.nakadi.security.Client;
+import org.zalando.nakadi.service.TopicsService;
+import org.zalando.problem.Problem;
+import org.zalando.problem.spring.web.advice.Responses;
+
+import javax.ws.rs.core.Response;
+
+@RestController
+@RequestMapping(value = "/topics/")
```

Comment:
I am missing security configuration for the new endpoint. (And the test for that)